lib/access_import/acc_utils.rb

```diff
@@ -0,0 +1,54 @@
+module AccUtils
+  def make_mappable(table_name, primary_key, sanitize = false)
+    @acdb[table_name].map do |r|
+      [
+        r[primary_key].to_i,
+        sanitize ? sanitize_record(r) : r
+      ]
+    end.to_h
+  end
+
+  def down_hkeys(row)
+    row.keys.map do |key|
+      [key.to_s.underscore.to_sym, row[key]]
+    end.to_h
+  end
+
+  def parse_int_fields(record, keys)
+    keys.each do |key|
+      record[key] = record[key].to_i if record[key]
+    end
+    record
+  end
+
+  def parse_datetime_fields(record, keys)
+    keys.each do |key|
+      record[key] = record[key].to_datetime if record[key]
+    end
+    record
+  end
+
+  def parse_date_fields(record, keys)
+    keys.each do |key|
+      record[key] = record[key].to_date if record[key]
+    end
```

Comment:
but how do I gain control on what keys are transformed with  `transform_values`? 

Because I'm not transforming all values in the hash, and that is definitely not the desired beaviour here.